install.sh

```diff
@@ -108,26 +155,26 @@ function install() {
         # ensure folders exists
         for SUBCONFIG_FOLDER in $SUBCONFIG_FOLDERS ; do
             SUBCONFIG_FOLDER=$(sanitizePath "$SUBCONFIG_FOLDER")
-            echo "creating '/usr/lib/systemd/$SERVICE/$SUBCONFIG_FOLDER'"
-            (cd "/usr/lib/systemd/" && cd "$SERVICE" && sudo mkdir -p "$SUBCONFIG_FOLDER")
+            echo "creating '$DEST_DIR/lib/systemd/$SERVICE/$SUBCONFIG_FOLDER'"
+            mkdir -p "$DEST_DIR/lib/systemd/$SERVICE/$SUBCONFIG_FOLDER"
         done
```

Comment:
You might have missed this by accident (it was in the patch at https://gist.github.com/icedream/567188afa09551e2bf63a83550406741#file-fixup-patch-L118-L119). This still does not create `$DEST_DIR/lib/systemd/$SERVICE`, only subfolders of it.

Reply:
It should now implicitly be created on demand with this line
`mkdir -p "$DEST_DIR/lib/systemd/$SERVICE/$SUBCONFIG_FOLDER"`

Reply:
@leopoldhub

> The script will fail next if lib/systemd/$SERVICE/$SUBCONFIG_FOLDER (lib/systemd/system/system-sleep) does not exist yet in the subconfigs loop right afterwards. The script has mkdir -p calls for the subconfig - but this one only runs if the subconfig provides subdirectories of its own which is currently never the case.

As I explained above, this `mkdir -p` call isn't even triggered because no subconfig has a subfolder. I tried to be clear about this with "only subfolders of it" but the code is generally confusing around this unfortunately.

Reply:
Sorry, I misunderstood
It is indeed confusing, I will have to refactor that part later on
It should be good now